src_c/circle.c

```diff
@@ -653,6 +673,10 @@ static PyGetSetDef pg_circle_getsets[] = {
     {"r", (getter)pg_circle_getr, (setter)pg_circle_setr, NULL, NULL},
     {"r_sqr", (getter)pg_circle_getr_sqr, (setter)pg_circle_setr_sqr, NULL,
      NULL},
+    {"d", (getter)pg_circle_getdiameter, (setter)pg_circle_setdiameter, NULL,
```

Comment:
I'm not sure about the diameter attribute having an alias (`d` and `diameter`). I think this is usually used when we deprecate something. We can keep this in for now, but let's think about removing them in a future PR.

Reply:
I'll answer this the way i answered in discord: 
```
also about the circle.diameter attribute i've aliased that because rect aliases its attributes as well
and could be useful to have both, hurts no one
so if one needs readability will go for "diameter", and if one wants compact code will go for "d" 
as long as the aliases are not confusing i think we are fine with them
```